src/test/TestAccount.cpp

```diff
@@ -153,6 +153,19 @@ TestAccount::manageData(std::string const& name, DataValue* value)
     }
 }
 
+
+void
+TestAccount::bumpSequence(AccountID const& account, std::vector<SecretKey>* pSigners, SequenceNumber to, BumpSeqValidRange* range)
+{
+    auto pTx = tx({txtest::bumpSequence(account, to, range)});
+    if (pSigners) {
+        for (auto& from : *pSigners) {
+            pTx->addSignature(from);
```

Comment:
I think it would be better to pass one SecretKey instead of AccountID and vector of SecretKeys here (or just another TestAccount instance).

Reply:
This is undesirable because certain things with bumpsequence that are desirable to test require 2 different signers on one account.

Reply:
For these cases I've used different pattern. The TestAccount::pay/merge/doanything methods are implemented to create transactions that pass initial validation.

For other ones I was building them manually like in https://github.com/stellar/stellar-core/blob/master/src/transactions/MergeTests.cpp#L76 (complex transaction with few operations) or like in https://github.com/stellar/stellar-core/blob/master/src/transactions/TxEnvelopeTests.cpp#L59 (executing transactions with invalid signatures).

I think we should keep it that way to not bloat TestAccount interface for non-common cases.